src/crypto/CHIPCryptoPAL.h

```diff
@@ -691,6 +692,12 @@ class Spake2p
     void * order;
     void * tempbn;
 
+    /*
+     *  Context = "SPAKE2+ Commissioning" || PBKDFParamRequest || PBKDFParamResponse
```

Comment:
Context is now Hash("CHIP PAKE V1 Commissioning" || PBKDFParamRequest || PBKDFParamResponse). Since the Spake2+ impl ingests the context during initialization, why do we need this buffer here ? Shouldn't this be attached to the SecurePairingSession ?

Reply:
Agree. Buffer can be moved to SecurePairingSession.

Reply:
moved buffer to PASESession